apps/activejobs/app/models/jobstatusdata.rb

```diff
@@ -201,6 +203,43 @@ def extended_data_pbspro(info)
     self
   end
 
+  # Store additional data about the job. (SGE-specific)
+  #
+  # Parses the `native` info function for additional information about jobs on SGE systems.
+  #
+  # @return [Jobstatusdata] self
+  def extended_data_sge(info)
+    return unless info.native
+    attributes = []
+    attributes.push Attribute.new "Cluster", self.cluster_title
+    attributes.push Attribute.new "Job Id", self.pbsid
+    attributes.push Attribute.new "Job Name", self.jobname
+    attributes.push Attribute.new "User", self.username
+    attributes.push Attribute.new "Account", self.account
+    attributes.push Attribute.new "Project", info.native[:accounting_id] unless info.native[:accounting_id].nil?
```

Comment:
accounting_id is a method on info

Reply:
info.accounting_id is the same as self.account; will remove